doc/user-guides/kernel-parameters.rst

```diff
@@ -459,8 +459,6 @@ In the current configuration, we will set
 ``i915.enable_initial_modeset=1`` in Service VM and
 ``i915.enable_initial_modeset=0`` in User VM.
 
-This parameter is not used on UEFI platforms.
```

Comment:
Does this mean that this parameter can (or should) now be used on UEFI platforms (such as UP2, NUCs, etc.)?

Reply:
My question is irrelevant, that parameter is obsolete and no longer exists, please dismiss it.